### ISSUE TITLE
Book 05: Sentence 054

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#35]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5afac842-2244-4cef-ac02-2caedbe68c1a)

```
# sent_id = Book_05_-_The_Omens.txt:54
# text = Auh in juhquj, ic muchichiuh: yxtecujcujpul coztic: auh injc tlatoa pipitzca, iuhqujn mapipitzoa: tocujleoa, tocujlechoa: no iaotequjoa qujtocaiotiaia.
# text[norm] = Auh in iuhqui, ic mochichiuh: ixtecuicuilpol coztic: auh inic tlatoa pipitzca, iuhquin mapipitzoa: tocuilehua, tocuilechoa: no yaotequihua quitocayotiaya.
# text[orig] = A·uh¶injuhquj·,·ic·muchichiuh·:·yxtecuj¶cujpul·coztic·:·auh·injctlatoa·pipitzca·,·iuh·qujn·mapipitzoa·:·tocujleoa·,·tocujle¶choa·:·noiaotequjoa·qujto·caiotiaia·.
# text[osp] = La forma deste animal aca en esta tierra, es, que son como los de españa: que tienen la barriga, y pecho blanca: y todo lodemas bermejo.
# references = A&D/170:4
# tagged = 100.00%
# lemmatised = 100.00%	
# analysed = 100.00%
# heads = 100.00%
# relations = 42.31%
1	Auh	auh	CCONJ	_	_	6	cc	_	_
2	in	in	DET|SCONJ	_	_	6	_	_	_
3	juhquj	iuhqui	ADV	_	_	6	advmod	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	ic	ic	ADV	_	_	6	_	_	_
6	muchichiuh	chichihua	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Past|VerbForm=Fin	0	root	_	_
7	:	:	PUNCT	_	_	8	punct	_	_
8	yxtecujcujpul	ixtecuicuilpol	NOUN	_	_	6	_	_	_
9	coztic	coztic	ADJ	_	_	8	_	_	_
10	:	:	PUNCT	_	_	14	punct	_	_
11	auh	auh	CCONJ	_	_	14	_	_	_
12	injc	inic	SCONJ	_	_	13	_	_	_
13	tlatoa	tlatoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	14	_	_	_
14	pipitzca	pipitzca	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	6	_	_	_
15	,	,	PUNCT	_	_	17	punct	_	_
16	iuhqujn	iuhquin	ADV	_	_	17	_	_	_
17	mapipitzoa	mapipitzoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	14	_	_	_
18	:	:	PUNCT	_	_	19	punct	_	_
19	tocujleoa	tocuilehua	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	14	_	_	_
20	,	,	PUNCT	_	_	21	punct	_	_
21	tocujlechoa	tocuilechoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	19	_	_	_
22	:	:	PUNCT	_	_	25	punct	_	_
23	no	no	ADV	_	_	25	_	_	_
24	iaotequjoa	yaotequihua	NOUN	_	State=Abs	25	_	_	_
25	qujtocaiotiaia	tocayotia	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	6	_	_	_
26	.	.	PUNCT	_	_	6	punct	_	_	
```

**Comment by ftyers** [comment]
López Austin has:
> Y en esta forma, así se atavía: cara manchada, carilla manchada, amarilla. Y así hace: chilla, como si chiflara "tocuileoa, tocuile­choa". También se llama capitán de guerra.

**Comment by ftyers** [comment]
Updated:

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/105f8305-5c18-4c26-8754-1bf2c5f58def)

Questions:
- What is the function of *yaotequihua* in *no yaotequihua quitocayotiaya*, `obj` or `xcomp` or something else, what is the valency of *tocayotia*?
- What are the POS of *tocuilehua* and *tocuilechoa*? `VERB` doesn't seem right?
- What is the relation between *tlatoa* and *pipitzca*?
- Is *ixtecuicuilpol coztic* really a clause?
- What is the structure of *in iuhqui, ic mochichiuh*

**Comment by Lguyogiro** [comment]
> What are the POS of tocuilehua and tocuilechoa? VERB doesn't seem right?

what's wrong with VERB? they are at least consistent with common verb endings (e.g. it would be one thing if they had like absolutive endings or something).

Malinal has tocuilehua listed as an intransitive verb meaning "to utter barbaric cries" [https://www.malinal.net/lexik/nahuatlT.html#TOCUILEHUA](https://www.malinal.net/lexik/nahuatlT.html#TOCUILEHUA)

There is also the agentive _TOCUILECHOANI_ which supports the verbhood or tocuilechoa

**Comment by Lguyogiro** [comment]
> What is the relation between tlatoa and pipitzca?

I guess either `conj` (it speaks and shrieks/whistles), `advcl` (it speaks shriekingly), or `parataxis` (it speaks, it whistles/shrieks)? The `advcl` analysis is maybe not very strongly supported, since there is no subordinator between the verbs...

**Comment by Lguyogiro** [comment]
> Is ixtecuicuilpol coztic really a clause?

not sure, I'm confused about ixtecuicuilpol...I thought _cuicuil_... is like "spotted" or "multicolored", maybe _ixtecuicuilpol_ is like "its ugly spotted face"? in which case we ould say this is a copular clause "its ugly spotted face is yellow".  since this isn't obviously subordinated or an argument of another verb, I think it is reasonable to analyze it as a clause.

**Comment by ftyers** [comment]
Update:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/6a7175a0-0b63-4e3d-b192-a6dea7d38eb6)

There is also here an *in iuhqui* and an *iuhquin* (see #13, #6)

**Comment by Lguyogiro** [comment]
update looks good.

> There is also here an in iuhqui and an iuhquin (see https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/issues/13, https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/issues/6)

Yeah I was jus thinking about this....you don't suppose _in iuhqui_ is an adverbial clause? 

Also, here is another use of _ic_, which seems to behave more like a `reln` in this case...I will look back at previous sentences to see if we've already handled this case.

**Comment by Lguyogiro** [comment]
> Like And being thus, like this it adorns itself ...

Yeah I like treting this as `advcl`

> For me, this feels like ADV, if it is RelN, what is the complement? :D

Yeah I don't know. Lockhart mentions its use being similar to 3rd person sing reln, "for which reason". But I could go either way.

**Comment by Lguyogiro** [comment]
LGTM :+1:

**Comment by ftyers** [comment]
Resolved in a5bed0b.
